### ISSUE TITLE
Loss function decreasing 

### ISSUE THREAD

**Issue by Mahmood1968** [dell-ai-engineering/BigDL-ImageProcessing-Examples#7]
The loss function never decrease or keeps fluctuating between two numbers . I tried to used different batch size , epoch and learning rate between 0.01 and 0.001. But not that much improvement . What suggestions that you have to tune these parameters . is there any other reason other than these parameters

**Comment by hhbyyh** [comment]
![image](https://user-images.githubusercontent.com/7981698/47039168-1fff5100-d138-11e8-9b54-b0c32d7a9b7c.png)

I used a small part of the images as training data (around 200 images) and got the above result. Please check:
1. In the tensorboard, you can adjust the smooth rate. Move it to 0 allows you to see the raw values.
2. Usually the loss will decrease in the first a few epochs, then it's likely to keep stable.
